README.md

```diff
@@ -12,6 +12,8 @@ It supports authentication via simple access tokens, JWT endpoints and GitHub Ap
 
 It is also very light on GitHub, doing as few requests as necessary.
 
+You can find this library in [maven central repository](https://mvnrepository.com/artifact/com.spotify/github-client).
```

Comment:
Should we alternatively move this into the `Getting Started` section?

Reply:
Indeed it would make much more sense, let me move it